test/shared/src/main/scala/zio/test/ZIOSpecAbstract.scala

```diff
@@ -62,14 +62,15 @@ abstract class ZIOSpecAbstract extends ZIOApp {
         self.runSpec.zipPar(that.runSpec)
 
       def spec: ZSpec[Environment with TestEnvironment with ZIOAppArgs with Scope, Any] =
-        self.spec + that.spec
+        self.aspects.foldLeft(self.spec)(_ @@ _) + that.aspects.foldLeft(that.spec)(_ @@ _)
```

Comment:
We're still doing an `aspects.foldLeft` down on line 146, but that might be okay, because it will only be folding in `aspects` from the composed spec, which won't contain these "sub aspects"